src/Stratis.SmartContracts.Executor.Reflection/Compilation/SmartContractDecompiler.cs

```diff
@@ -35,6 +35,9 @@ public static SmartContractDecompilation GetModuleDefinition(byte[] bytes, IAsse
                 result.ContractType = developedTypes.FirstOrDefault(x => x.CustomAttributes.Any(y => y.AttributeType.Name == typeof(DeployAttribute).Name));
             }
 
+            // If contract type wasn't set yet, set the first.
```

Comment:
Still working out the exact right functionality here - whether we want to throw an exception or just work around it and deploy the first one. 

Will be solved in https://github.com/stratisproject/StratisBitcoinFullNode/issues/1752